src/System.Net.Security/src/System/Net/SecureProtocols/_FixedSizeReader.cs

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+namespace System.Net
+{
+    //
+    // Reads a fixed size packet from a stream, can disover EOF as 0 bytes read from stream.
+    //
+    internal class FixedSizeReader
+    {
```

Comment:
> Most of current code in CoreFx doesn't appear to use files with underscores in them. I wonder if we have any guidance of naming like this?

We've tried to remove underscores from file names in cases like this.  It's possible some slipped through or we just didn't finish cleaning everything up.